backend/migrations/dataall_migrations/base_migration.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import os
+from abc import ABC, abstractmethod
+from typing import Type
+
+logger = logging.getLogger()
+logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
+
+
+class MigrationBase(ABC):
+    @staticmethod
+    @abstractmethod
+    def revision_id() -> str:
+        """
+        Uniq revision identifier.  To be implemented in the  inherited classes
```

Comment:
nit: I'd remove the "To be implemented.." comment, it's obvious from the `@abstractmethod`